snakemake/deployment/conda.py

```diff
@@ -211,6 +212,29 @@ def create_archive(self):
             raise e
         return env_archive
 
+    def execute_deployment_script(self, env_file):
+        """Execute post-deployment script if present"""
+        from snakemake.shell import shell
+
+        post_deploy_script = Path(env_file).with_suffix(".post-deploy.sh")
```

Comment:
Having thought about this, it is actually not that easy. env_file can be a remote URL. We need to handle the two cases in line 246 below, and also store the post deploy script locally in case it is present.

Reply:
This should be done now.